### DISCUSSION TITLE
what does RSS button on the interface do?

### DISCUSSION THREAD

**Discussion by awg13** [FreshRSS/FreshRSS#5710] [Q&A]
If I get the URL of this, it shows as ```https://freshrss.example.com/i/?a=rss&rid=652baa34ad030&hours=168```

So does that mean if I get the RSS feed from there, it will expire after 168 hours?

**Comment by Alkarex** [comment]
The parameters control the number of entries per RSS feed (see https://github.com/FreshRSS/FreshRSS/issues/1249)

* Since X hours: `https://freshrss.example/i/?a=rss&hours=3`
* Explicit number: `https://freshrss.example/i/?a=rss&nb=10`
* Limited by `min_posts_per_rss` and `max_posts_per_rss` in user config
